tensorboard/components/experimental/plugin_lib/tf-plugin-lib.ts

```diff
@@ -13,8 +11,7 @@
 WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
--->
-<link rel="import" href="../tf-plugin-util/message.html" />
-<script src="plugin-guest.js"></script>
+==============================================================================*/
+import * as _runs from './runs.js';
```

Comment:
is the file extension important?

Reply:
The way I've written the tests, I'm importing modules which need the ".js" to fetch properly.
I think there could be a way to make the tests work without the extension, but I would like to defer that to future investigation.

Reply:
That's fair. :+1: